aten/src/ATen/native/NonSymbolicBC.h

```diff
@@ -22,5 +22,7 @@ TORCH_API at::Tensor _embedding_bag_sparse_backward(const at::Tensor & grad, con
 TORCH_API at::Tensor value_selecting_reduction_backward(const at::Tensor & grad, int64_t dim, const at::Tensor & indices, at::IntArrayRef sizes, bool keepdim);
 TORCH_API at::Tensor trace_backward(const at::Tensor & grad, at::IntArrayRef sizes);
 TORCH_API at::Tensor index_select_backward(const at::Tensor & grad, at::IntArrayRef self_sizes, int64_t dim, const at::Tensor & index);
+TORCH_API at::Tensor select(const at::Tensor& self, int64_t dim, int64_t index);
+TORCH_API at::Tensor select_backward(const at::Tensor& grad, at::IntArrayRef input_sizes, int64_t dim, int64_t index);
```

Comment:
Used by XLA: https://github.com/pytorch/xla/blob/master/torch_xla/csrc/aten_xla_type.cpp

Reply:
This is a mistake since these are not exposed externally. Instead, I need to submit a PR to XLA with SymInt stuff. On it.